Advent24 day14: Page by Page: How Pagination Makes the Web Accessible

miniemuff opened the pull request:
my article for your advent calendar 2024, day 14.
title: Page by Page: How Pagination Makes the Web Accessible

I'm happy about feedback.

matuzo commented:
Hey Kristin,

thanks a lot for your great post. We've added some comments directly in your file in the advent24 branch of the HTMHell repository. Sorry that we're not using GitHub and its comment feature as intended but our approach makes it easier for us since not all authors are using GitHub.

Please review our comments, make your changes if necessary, and submit another PR. Your post goes online in 19 days. It would be great if you could submit your changes a couple of days earlier. If you have questions please ask them here.

Thanks a lot,
Manuel, Saptak, and Karl

miniemuff commented:
hello,thank you very much for reviewing my post. i read through your feedback very carefully and have now reviewed it.I have just opened the pull request: https://github.com/matuzo/HTMHell/pull/184I really appreciate about your feedback. I currently have the time to include your feedback as soon as possible.Thank you for letting me be part of your advent calendar,Kristinps: i hope the formatting of the post didn't break because i added the changes via the github interface.Am 25.11.2024 um 20:20 schrieb Manuel Matuzovic ***@***.***>:﻿
Hey Kristin,
thanks a lot for your great post. We've added some comments directly in your file in the advent24 branch of the HTMHell repository. Sorry that we're not using GitHub and its comment feature as intended but our approach makes it easier for us since not all authors are using GitHub.
Please review our comments, make your changes if necessary, and submit another PR. Your post goes online in 19 days. It would be great if you could submit your changes a couple of days earlier. If you have questions please ask them here.
Thanks a lot,
Manuel, Saptak, and Karl

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
